nixos/release-combined.nix

```diff
@@ -106,8 +116,8 @@ in rec {
         (all nixos.tests.xfce)
 
         nixpkgs.tarball
-        (all nixpkgs.emacs)
-        (all nixpkgs.jdk)
+        (all allSupportedNixpkgs.emacs)
+        (all allSupportedNixpkgs.jdk)
```

Comment:
Given my comment here: https://github.com/NixOS/nixpkgs/pull/27923#discussion_r131421251 I'll expand:

My intention wasn't for any of the blocking packages to change. ie: `tested` remains completely unchanged. The major change is in not fanning out and building the whole package set.

If this isn't what we want that is okay, but to keep this the same, I think the code here does that correctly.